### DISCUSSION TITLE
Command palette overrides browser shortcut

### DISCUSSION THREAD

**Discussion by q--** [community/community#14762] [Other Feature Feedback, Questions, & Ideas] [answered]
I always use Ctrl+K to go to the Firefox search bar, I don't want the command palette (whatever that is) to take it over so I disabled the feature.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @q--,  we have a feature that allows you to customize the keyboard shortcuts used to open the command palette. You can edit your shortcuts in the [Accessibility section](https://github.com/settings/accessibility) of your user settings.

**Reply by mkp05** [reply]
It's nice, but personally I would prefer approach I've seen in some other services - when I use the overridden shortcut for the second time in very short period of time - it fallbacks to default browser functionality. With current "accessibility" approach I need to add additional functional keys which is not as convenient.

**Reply by mitchellmebane** [reply]
In addition, editing the shortcuts in the accessibility section of user settings only works when I am logged in.

**Reply by StefanWritingSoftware** [reply]
I was in the same boat as the original poster - relying on `k` for the Firefox search bar.
This post initially gave me hope, but we are given only the choice between `k` and `p` as modifier keys.  The former being in conflict with browser search, the latter in conflict with printing or opening private browser windows.

If offering overrides, please consider keys that are not part of major browser or OS default hotkey configurations!

Even text editing fields are in conflict with the current offering -- command-k, when not bound to the Command Palette, is captured and converted to a (pretty darn annoying) template for entering a markdown link.
